clang-tools-extra/clang-tidy/cppcoreguidelines/MissingStdForwardCheck.cpp

```diff
@@ -79,6 +79,8 @@ AST_MATCHER_P(LambdaExpr, hasCaptureDefaultKind, LambdaCaptureDefault, Kind) {
   return Node.getCaptureDefault() == Kind;
 }
 
+AST_MATCHER(ParmVarDecl, hasAnyName) { return !Node.getName().empty(); }
```

Comment:
use just [getIdentifier](https://clang.llvm.org/doxygen/classclang_1_1NamedDecl.html#a1475b4e0cfc82be122ba166894f68705)() as getName got assert inside, and change name, simply because we have matcher of such name already, so maybe something like `hasIdentifier`